### PR TITLE
🐛Adds capability for closing story share menu on escape key press.

### PR DESCRIPTION
**Pull request by johannaperez**
Improves accessibility by allowing users to close the menu on escape keypress
Fixes 22747
@newmuis

### CODE REVIEW

File: extensions/amp-story/1.0/amp-story-share-menu.js [hunk 1]
```diff
@@ -221,13 +229,12 @@ export class ShareMenu {
...
    * Handles click events and maybe closes the menu for the fallback UI.
    * @param  {!Event} event
-   * @return {*} TODO(#23582): Specify return type
    */
   onShareMenuClick_(event) {
     const el = dev().assertElement(event.target);
 
     if (el.classList.contains('i-amphtml-story-share-menu-close-button')) {
-      return this.close_();
+      this.close_();
```
Comment (newmuis) [inline extensions/amp-story/1.0/amp-story-share-menu.js]: Looks like there was a PR that got merged in the meantime.  Can you update to add back the return and its jsdoc?

Comment (johannaperez) [reply extensions/amp-story/1.0/amp-story-share-menu.js]: Hmmm I actually deleted those lines as they weren't needed. Should I remove them in a sep PR?

Comment (newmuis) [reply extensions/amp-story/1.0/amp-story-share-menu.js]: Whoops my bad, you're absolutely right!  Thought this was related to #23559, but (a) it's not, and (b) that's been merged for a while now.

**Comment by johannaperez** [comment]
Fixes #22747

**Comment by newmuis** [comment]
FYI @johannaperez, we use Prettier to format code, and it has found a few nits that need to be changed: https://travis-ci.org/ampproject/amphtml/jobs/568555973

**Comment by johannaperez** [comment]
Got it. Will look at linter checks and update.

**Pull request merged**